tests/integration/errors/test_error_handler_false.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should capture and report errors along with a stack trace 
+when a custom error handler exists but it returns false for a handled
+error.
+*/
+
+/*SKIPIF
+<?php
+if (version_compare(PHP_VERSION, "7.0", "<")) {
+  die("skip: PHP < 7.0.0 not supported\n");
+}
+*/
+
+/*INI
+display_errors=1
+log_errors=0
+*/
+
+/*EXPECT_REGEX
+^\s*(PHP )?Deprecated:\s*Let this serve as a deprecation*
```

Comment:
Just an educational comment: with `INI` being `display_errors=1` and `log_errors=0` the `(PHP )?` part can be skipped from expected output.